wrappers/angular/README.md

```diff
@@ -91,7 +108,8 @@ import { HotTableModule } from '@handsontable/angular';
 export class AppModule { }
 ```
 
-**Angular Component**
+#### Angular Component
```

Comment:
@adrianspdev I wonder if we can purely show a component similar to the suggestions for React? as the user shouldn't need to create a whole app to use our framework.